js/index.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+	$('form').submit(addEntry);
+
+	function addEntry (event) {
+		event.preventDefault();
+
+	var input = $('#city-type').val();
+
+		if(input === 'San Fransisco') {
+			$('body').addClass('.sf');
```

Comment:
I know this is a common source of confusion. The stuff that you put inside the `$('...')` you treat like a CSS selector. Therefore, you want to select something on the page by its class name you do `.class-name` (e.g. `$('.sf')`)

However, whenever you are using jQuery's `addClass`, `toggleClass`, or `removeClass` methods, you omit the `.`. Otherwise in this case, what would end up happening is the body tag would rendered like this:
```
<body class=".sf">
...
</body>
```

When in fact what you want is simply this:
```
<body class="sf">
...
</body>
```

Hope that helps clarify! :)